docs/eventing/03-03-in-cluster-eventing.md

```diff
@@ -0,0 +1,51 @@
+---
+title: In-cluster eventing
+type: Details
+---
+
+Kyma in-cluster Eventing allows publishers to send messages and subscribers to receive them without the need for a Kyma Application. This means that instead of the usual event flow where Application Connector publishes events to the NATS Publisher Proxy, events can be published from within the cluster.
```

Comment:
```suggestion
Kyma in-cluster Eventing allows publishers to send messages and subscribers to receive them without the need for a Kyma Application. This means that instead of the usual event flow where Application Connector publishes events to the NATS Publisher Proxy, events can be published from within the cluster directly to the NATS Publisher Proxy.
```